sentry_sdk/utils.py

```diff
@@ -209,39 +233,47 @@ def should_hide_frame(frame):
 
 
 def iter_stacks(tb):
+    # type: (Any) -> Iterator[Any]
     while tb is not None:
         if not should_hide_frame(tb.tb_frame):
             yield tb
         tb = tb.tb_next
 
 
 def slim_string(value, length=512):
+    # type: (str, int) -> str
     if not value:
         return value
     if len(value) > length:
         return value[: length - 3] + "..."
     return value[:length]
 
 
-def get_lines_from_file(filename, lineno, loader=None, module=None):
+def get_lines_from_file(
+    filename,  # type: str
+    lineno,  # type: int
+    loader=None,  # type: SourceFileLoader
+    module=None,  # type: str
+):
+    # type: (...) -> Tuple[List[str], Optional[str], List[str]]
     context_lines = 5
     source = None
-    if loader is not None and hasattr(loader, "get_source"):
+    if loader is not None and module is not None and hasattr(loader, "get_source"):
```

Comment:
Sure `module` is necessary? I think some loaders are module specific and ignore the first argument to `get_source`.

Reply:
I reverted it, typehints on stdlib are broken